### PR TITLE
chore: manual QA checklist

### PR DESCRIPTION
**Pull request by olevski**
~This is still a WIP and a draft.~

Initial QA checklist.

### CODE REVIEW

File: qa/README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  <details>
+    <summary>Details</summary>
+    
+    1. Pick a template
+    2. Create a project
+    3. Wait for the image to build
+    4. Ensure the image built
+    5. Start a session
+    6. Ensure the session can be accessed
+    7. Repeat for all templates
```
Comment (rokroskar) [inline qa/README.md:120]:
```suggestion
    1. Pick a template
    2. Create a project
    3. Wait for the image to build
    4. Ensure the image built
    5. Start a session
    6. Ensure the session can be accessed
    7. Ensure the renku CLI is installed and working
    8. Repeat for all templates
```

File: qa/README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+## Projects
+
+- [ ] Create a new project on Renku (UI)
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new project in the website
+    2. Ensure the KG integration is active
+    3. Ensure a session can be successfully started on the project
+    4. Ensure said sesssion can be accessed
```
Comment (rokroskar) [inline qa/README.md:19]:
```suggestion
    4. Ensure said sesssion can be accessed
    5. Ensure the Renku CLI is there
    6. Modify one of the environment files (e.g. `requirements.txt`) and run `renku save`
    7. Shut down the session, launch a new one from the new commit and make sure the package you added is there
```

File: qa/README.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+## Datasets
+
+- [ ] Create a dataset from the UI
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new Renku project in the website
+    2. Download a dataset from https://zenodo.org/
+    3. Upload the data and create the dataset from the UI
```
Comment (rokroskar) [inline qa/README.md:53]: If we are downloading data from zenodo, why not test the import functionality? 

If testing dataset creation with uploaded data, just find some cat photos online and upload them

File: qa/README.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    9. Run `renku dataset ls` can you see the dataset you uploaded?
+  </details>
+
+- [ ] Create a dataset with the CLI from a running session
+  <details>
+    <summary>Details</summary>
+
+    1. Start a renku session
+    2. Navigate to https://zenodo.org/
+    3. Pick a random dataset and import it in your session with `renku dataset import <doi-from-zenodo>`
```
Comment (rokroskar) [inline qa/README.md:68]:
```suggestion
    3. Pick a random dataset and import it in your session with `renku dataset import <zenodo-url>`
```

File: qa/README.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    4. Run `git push`
+    5. Ensure the dataset was downloaded successfully
+    6. Are large files checked in LFS?
+    6. Navigate to the "Datasets" section of the project on the website, can you see the dataset you imported?
+    7. Is the
```
Comment (rokroskar) [inline qa/README.md:73]:
```suggestion

```

File: qa/README.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    4. Run `git push`
+    5. Ensure the dataset was downloaded successfully
+    6. Are large files checked in LFS?
+    6. Navigate to the "Datasets" section of the project on the website, can you see the dataset you imported?
+    7. Is the 
+  </details>
+
+## Sessions
+
+- [ ] Autosaves
```
Comment (rokroskar) [inline qa/README.md:78]: do we want this here now? Hopefully it will change in the next ~2 sprints?

**Review by rokroskar — CHANGES_REQUESTED** [review]
This is a great start! Some minor comments

File: qa/README.md (hunk 3 above)
Comment (olevski) [reply qa/README.md:53]: I was thinking that the data found on Zenodo is more representative of a research/data science dataset. A few cat photos are small and easy to handle usually. I think striving to handle any Zenodo file is a good thing. I imagine once in a while someone will pick a file that is too large or that causes problems. With cat photos it is more likely to be trivial. But yeah cat photos is fine too. Dogs too 😸

File: qa/README.md (hunk 6 above)
Comment (olevski) [reply qa/README.md:78]: I will remove it Rok. I was thinking the same thing that maybe we should not add it at all.

File: qa/README.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+## Datasets
+
+- [ ] Create a dataset from the UI
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new Renku project in the website
+    2. Download a dataset from https://zenodo.org/
+    3. Upload the data and create the dataset from the UI
+    4. Ensure that the download completed
```
Comment (Panaetius) [inline qa/README.md:57]:
```suggestion
    4. Ensure that the upload completed
```

since above we say "upload the data" ?

File: qa/README.md [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+- [ ] Create a workflow file in a session
+  <details>
+    <summary>Details</summary>
+    
+    To be filled in
+  </details>
+  
+## Templates
+
+- [ ] Create a project and start a session
```
Comment (Panaetius) [inline qa/README.md:89]: What's the difference between this and `Create a new project on Renku (UI)` higher up?

File: qa/README.md (hunk 3 above)
Comment (rokroskar) [reply qa/README.md:56]: yeah maybe - but then if we're just going to zenodo to get some data (and not using the zenodo import functionality) then we should just put some large-ish files somewhere for people to grab. Finding something usable/useful in zenodo can be hit-or-miss in my experience.

File: qa/README.md (hunk 8 above)
Comment (bethcg) [reply qa/README.md:89]: @Panaetius, I understand the difference with the higher up is that here we aim to test all available default templates work properly (line 100) (which makes sense to me).

Comment (olevski) [reply qa/README.md:89]: The point is to test all templates that have not been touched with the other tests above. I will clarify.

File: qa/README.md [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+- [ ] Create a dataset from the UI
+  <details>
+    <summary>Details</summary>
+
+    1. Create a new Renku project in the website
+    2. Download a dataset from https://zenodo.org/
+    3. Upload the data and create the dataset from the UI
+    4. Ensure that the download completed
+    5. Ensure that large files are in Git LFS
+    6. After uploading the dataset try to clone the git project locally (does this take a really long time)
```
[the remark this answers is not in the document]
Comment (olevski) [reply qa/README.md:59]: It is easier to focus the tests on the web ui. We can add additional tests of the CLI. Mixing the two in the same tests can be confusing.

File: qa/README.md (hunk 7 above)
Comment (olevski) [reply qa/README.md:57]: I fully reworded this seciont

**Review by bethcg — DISMISSED** [review]

**Review by rokroskar — APPROVED** [review]

**Pull request merged**